ww-config.js

```diff
@@ -186,6 +186,7 @@ export default {
                                 label: { en: 'Sortable' },
                                 type: 'OnOff',
                                 defaultValue: false,
+                                hidden: true,
```

Comment:
Why ?

Reply:
Flo asked to hide it for now as it's not working properly

Reply:
We need to merge the new Path Input before